main.js

```diff
@@ -0,0 +1,101 @@
+let beginBoard;
+let startButton = document.getElementById('start')
+let status = document.getElementById('status')
+
+startButton.addEventListener('click', function() {
+    event.target.disabled = true;
+    event.target.textContent = "Player X's turn"
+})
+
+    const humanPlayer = 'X';
+    const computerPlayer = 'O';
+
+    const winConditions = [
+        [0, 1, 2], // row
+        [3, 4, 5], // row
+        [6, 7, 8], // row
+        [0, 3, 6], // column
+        [1, 4, 7], // column
+        [2, 5, 8], // column
+        [0, 4, 8], // diagonal
+        [6, 4, 2]  // diagonal
+    ]
+
+const cells = document.querySelectorAll ('.cell'); // .cell or .board?
```

Comment:
This will work if you give all of your cells a classname 'cell' in your html. Currently your cells only have an ID.